wombat/src/main/cpp/subsystems/Arm.cpp

```diff
@@ -3,66 +3,74 @@
+  table->GetEntry("initialAngle")
+      .SetDouble(initialAngle.convert<units::degree>().value());
+  table->GetEntry("angleOffset")
+      .SetDouble(initialAngle.convert<units::degree>().value());
 }
 
-//arm config is used
-Arm::Arm(ArmConfig config)
-  : _config(config),
-    _pid(config.path + "/pid", config.pidConfig),
-    _velocityPID(config.path + "/velocityPID", config.velocityConfig),
-    _table(nt::NetworkTableInstance::GetDefault().GetTable(config.path))
-{
-}
+// arm config is used
+wom::subsystems::Arm::Arm(wom::subsystems::ArmConfig config)
+    : _config(config), _pid(config.path + "/pid", config.pidConfig),
+      _velocityPID(config.path + "/velocityPID", config.velocityConfig),
+      _table(nt::NetworkTableInstance::GetDefault().GetTable(config.path)) {}
 
-//the loop that allows the information to be used
-void Arm::OnUpdate(units::second_t dt) {
-  //sets the voltage and gets the current angle
+// the loop that allows the information to be used
+void wom::subsystems::Arm::OnUpdate(units::second_t dt) {
+  // sets the voltage and gets the current angle
   units::volt_t voltage = 0_V;
   auto angle = GetAngle();
 
-  //sets usable infromation for each state
+  // sets usable infromation for each state
   switch (_state) {
-    case ArmState::kIdle:
-      break;
-    case ArmState::kVelocity:
-      {
-        units::newton_meter_t torque = 9.81_m / 1_s / 1_s * _config.armLength * units::math::cos(angle + _config.angleOffset) * (0.5 * _config.armMass + _config.loadMass);
-        // units::volt_t feedforward = _config.leftGearbox.motor.Voltage(torque, 0_rad/1_s);
-        units::volt_t feedforward = _config.leftGearbox.motor.Voltage(torque, _velocityPID.GetSetpoint());
-        // feedforward = 3.5_V;
-        // std::cout << "feedforward" << feedforward.value() << std::endl;
-        voltage = _velocityPID.Calculate(GetArmVelocity(), dt, feedforward);
-        // std::cout << "arm velocity voltage is: " << voltage.value() << std::endl;
-        // voltage = 0_V;
-      }
-      break;
-    case ArmState::kAngle:
-      {
-        units::newton_meter_t torque = 9.81_m / 1_s / 1_s * _config.armLength * units::math::cos(angle + _config.angleOffset) * (0.5 * _config.armMass + _config.loadMass);
-        units::volt_t feedforward = _config.leftGearbox.motor.Voltage(torque, 0_rad/ 1_s);
-        // std::cout << "feedforward" << feedforward.value() << std::endl;
-        voltage = _pid.Calculate(angle, dt, feedforward);
-      }
-      break;
-    case ArmState::kRaw:
-      voltage = _voltage;
-      break;
+  case wom::subsystems::ArmState::kIdle:
+    break;
+  case wom::subsystems::ArmState::kVelocity: {
+    units::newton_meter_t torque =
+        9.81_m / 1_s / 1_s * _config.armLength *
+        units::math::cos(angle + _config.angleOffset) *
+        (0.5 * _config.armMass + _config.loadMass);
+    // units::volt_t feedforward = _config.leftGearbox.motor.Voltage(torque,
```

Comment:
remove commented out code

Reply:
Why was it commented out in the first place?

Reply:
ive applied a slightly different fix